### PR TITLE
Delegate max_query_parameters config to katsu

### PR DESCRIPTION
**Pull request by brouillette**
- bento_public now avoids duplicate configuration and fetches the `max_query_parameters` field from katsu when needed

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -190,22 +239,33 @@ func main() {
...
-		return katsuRequest("/api/public_overview", nil, c, func(j JsonLike) JsonLike {
-			// Wrap the response from Katsu in {overview: ...} (for some reason)
-			return JsonLike{"overview": j}
-		})
+		// restore from cache if present/not expired
+		publicOverview, err := getKatsuPublicOverview(c, katsuCache)
+		if err != nil {
+			return internalServerError(err, c)
+		}
+		fmt.Println("for fun")
```
Comment (davidlougheed) [inline main.go:267]: rm

**Review by davidlougheed — CHANGES_REQUESTED** [review]
this adds some empty files in `www/public`, what are those for?

File: main.go (hunk 1 above)
Comment (brouillette) [reply main.go]: whoopsies

**Review by v-rocheleau — APPROVED** [review]
lgtm

**Pull request merged**